### ISSUE TITLE
[suggestion] about flurry

### ISSUE THREAD

**Issue by matrixreal** [finscn/Ejecta#26]
@finscn 
now this fork is better than ever ... it has everything and the encryption tool is awesome
the last thing i can ask to make this fork best ever ever ever is flurry or google analytics
i think it's not hard since there are only 2 events (start time + end time)
hope that will be possible

**Comment by finscn** [comment]
What's flurry ?

**Comment by matrixreal** [comment]
flurry analytics like google analytics
https://developer.yahoo.com/flurry/#get-started=
i think there are not a lot of lines to be done

**Comment by matrixreal** [comment]
its not the ads parts but the analytics parts

**Comment by finscn** [comment]
I hope you could integrate them by yourself.
Because they are not core function of Game Develop , and I don't want to integrate too many SDKs in Ejecta.

**Comment by matrixreal** [comment]
that's okay thanks

**Comment by finscn** [comment]
@matrixreal , I suggest you choose some analytics service via pure JS (use ajax send/post data to service server) .

**Comment by matrixreal** [comment]
@finscn 
don't find analytics service via pure js
can you help me on adding flurry analytics please ?

**Comment by neodraw** [comment]
I agree that with some sort of analytics would be better. Google and Flurry does not work out of the box. If Flurry sdk is too complicated or large, maybe gameanalytics ( http://www.gameanalytics.com/) will be simplier for integration
